components/AdminFunc/CreateClass.tsx

```diff
@@ -26,7 +26,6 @@ export default function CreateClass() {
     e.preventDefault()
     try {
       await createSportClass()
```

Comment:
You are 100% sure that this action ALWAYS goes correctly?
So no chance that your server might go offline, internet connections might fail or some sorts?

If so, you are always 100% sure (which I wouldn't be)
you should catch the response and do stuff based on that response (always)

```
const response = await createSportClass();
// e.g.
const statusCode = await response.status;
```

Reply:
the displayerror component catches any error that comes from this mutation and will show it to the user.